lib/features/base/base_controller.dart

```diff
@@ -243,9 +243,6 @@ abstract class BaseController extends GetxController
         FcmConfiguration.initialize();
         FcmInteractorBindings().dependencies();
         FcmMessageController.instance.initializeFromAccountId(accountId, session);
-        if (!BuildUtils.isWeb) {
-          LocalNotificationManager.instance.setUp();
-        }
```

Comment:
Why remove it?

Reply:
I move it to `main()` for init with workmanager